### ISSUE TITLE
Clarify details in the docs for flashArrays

### ISSUE THREAD

**Issue by jayunit100** [purestorage/helm-charts#216]
@sdizer to add details ... 

```
flasharray:
  sanType: ISCSI
  defaultFSType: xfs
  defaultFSOpt: "-q"
  defaultMountOpt:
    - discard
  preemptAttachments: "true"
  iSCSILoginTimeout: 20
  iSCSIAllowedCIDR: ""

```

seems to be out of sync with the docs...

**Comment by jayunit100** [comment]
(xref https://github.com/purestorage/helm-charts/blob/master/pure-csi/values.yaml)

**Comment by sdodsley** [comment]
@jayunit100 / @sdizer Can you provide more details as to what is out of sync with what?

**Comment by sdodsley** [comment]
I suspect this has already been resolved by #215
